web/react/stores/user_store.jsx

```diff
@@ -64,12 +65,12 @@ var UserStore = assign({}, EventEmitter.prototype, {
   },
   setCurrentId: function(id) {
     if (id == null)
-      sessionStorage.removeItem("current_user_id");
+      localStorage.removeItem("current_user_id");
```

Comment:
ah...cool makes sense.  I wonder if this 1 special case should be abstracted into BrowserStore too (0/5). I like the direction of BrowserStore and if we can keep all the localStorage/sessionStorage code there then this can potentially make fixing stuff like "Safari Private Browsing Not Working" a lot easier.